static/themes/dark-theme.css

```diff
@@ -415,3 +415,11 @@ div.argmenuitem span.argdescription {
 .logo-sec {
     fill: #999999 !important;
 }
+
+textarea.tool-stdin {
```

Comment:
As with default-theme.css, layout should go into `static/explorer.css`

Reply:
ahhh ok makes sense, thanks